src/optimize/BaseOptimizer.js

```diff
@@ -74,7 +75,7 @@ class BaseOptimizer {
         path: this.env.workingDir,
         filename: '[name].bundle.js',
         sourceMapFilename: '[file].map',
-        publicPath: '/bundles/',
+        publicPath: this.urlBasePath + '/bundles/',
```

Comment:
This should probably have protection against `undefined`.  Something like what you did in uiApp.js.

```js
(this.urlBasePath || '') + '/bundles/'
```